src/components/cc-tile-metrics/cc-tile-metrics.smart.js

```diff
@@ -0,0 +1,92 @@
+import { ONE_DAY } from '@clevercloud/client/esm/with-cache.js';
+import { getAppMetrics, getGrafanaOrganisation } from '../../lib/api-helpers.js';
+import { defineSmartComponent } from '../../lib/define-smart-component.js';
+import { sendToApi } from '../../lib/send-to-api.js';
+import '../cc-smart-container/cc-smart-container.js';
+import './cc-tile-metrics.js';
+
+defineSmartComponent({
+  selector: 'cc-tile-metrics',
+  params: {
+    apiConfig: { type: Object },
+    ownerId: { type: String },
+    appId: { type: String },
+    grafanaBaseLink: { type: String },
+    consoleBaseLink: { type: String },
+  },
+  onContextUpdate ({ component, context, onEvent, updateComponent, signal }) {
+
+    const { apiConfig, ownerId, appId, grafanaBaseLink, consoleBaseLink } = context;
+
+    updateComponent('metrics', { state: 'loading' });
+
+    const metricsAppLink = getMetricsLink({ ownerId, appId, consoleBaseLink });
+    updateComponent('metricsLink', metricsAppLink);
+
+    fetchMetrics({ apiConfig, ownerId, appId, signal })
+      .then(({ cpuData, memData }) => {
+        if (cpuData.length === 0 || memData.length === 0) {
+          updateComponent('metrics', { state: 'empty' });
+        }
+        else {
+          updateComponent('metrics', { state: 'loaded', value: { cpuData, memData } });
+        }
+      })
+      .catch((error) => {
+        console.error(error);
+        updateComponent('metrics', { state: 'error' });
+      });
+
+    fetchGrafanaAppLink({ apiConfig, ownerId, appId, grafanaBaseLink, signal })
+      .then((grafanaAppLink) => {
+        updateComponent('grafanaLink', grafanaAppLink);
+      })
+      .catch((error) => {
+        console.error(error);
+        // TODO: find a way to remove console logic from this
+        const pathname = ownerId.startsWith('user_')
```

Comment:
Maybe a redirection logic inside the console may help us not have to do that here.

Reply:
Yes, I wonder how we can properly avoid this and a redirection would be a solution.